src/explore-education-statistics-admin/src/pages/release/data/components/__tests__/ReleaseDataUploadsSection.test.tsx

```diff
@@ -744,8 +762,11 @@ describe('ReleaseDataUploadsSection', () => {
     });
 
     test('successful submit with ZIP file renders with uploaded data file appended to list', async () => {
```

Comment:
@ntsim the changes to these zip file tests aren't related to the cancel button, I just noticed there was a duplicate key error when they were run and ended spending a while fixing them.

Reply:
Nice one! :smile: